standards/groundhog_standard.md

```diff
@@ -0,0 +1,235 @@
+
+
+# Specification
+
+## Summary
+A standardized interface for interacting with recurring subscriptions.
+
+##  Abstract
+Groundhog believes that the creation of a subscription should have a defined set of criteria, 
+we should be able to define a standardized public method that ensures that a subscription 
+can be created from any type of user interface without worry about specific implementation requirements.
+
+## Our Thoughts
+
+Interoperability between implementaitons should be the highest of priorties when it comes to developing the specification,
+abstract enough that we aren't bound by the needs of others, but specific enough that we aren't fragmented when it comes to how the specification gets utilized.
+
+## Motivation
+Recurring payments are the bedrock of SaSS and countless other businesses, a robust specification for defining this interaction will enable a broad spectrum of revenue generation and business models.
+
+#### Public View Functions
+
+
+
+
+###### isValidSubscription
+```SOLIDITY
+
+/** @dev Checks if the subscription is valid.
+  * @param bytes _subscriptionHash is the identifier of the customer's subscription with its relevant details.
+  * @return success is the result of whether the subscription is valid or not.
+*/
+
+function isValidSubscription(bytes _subscriptionHash) view public returns (bool success);
+```
+
+
+
+
+###### getSubscription
+```SOLIDITY
+
+/** @dev handles the incoming data to define subscription rules
+  * @param bytes _subscriptionHash is the identifier of the customer's subscription with its relevant details.
+  * @return (
+ address destination,
+ address recipient,
```

Comment:
should we add `token_address` and `token_decimals` as return items to this array?